docs/ops.html

```diff
@@ -4026,13 +4026,14 @@ <h4 class="anchor-heading"><a id="tiered_storage_config" class="anchor-link"></a
 
 <h5 class="anchor-heading"><a id="tiered_storage_config_broker" class="anchor-link"></a><a href="#tiered_storage_config_broker">Broker Configurations</a></h5>
 
-<p>By default, Kafka server will not enable tiered storage feature. <code>remote.log.storage.system.enable</code>
+<p>By default, the Kafka server will not enable the tiered storage feature. <code>remote.log.storage.system.enable</code>
   is the property to control whether to enable tiered storage functionality in a broker or not. Setting it to "true" enables this feature.
 </p>
 
 <p><code>RemoteStorageManager</code> is an interface to provide the lifecycle of remote log segments and indexes. Kafka server
-  doesn't provide out-of-the-box implementation of RemoteStorageManager. Configuring <code>remote.log.storage.manager.class.name</code>
-  and <code>remote.log.storage.manager.class.path</code> to specify the implementation of RemoteStorageManager.
+  doesn't provide out-of-the-box implementation of RemoteStorageManager. Users must configure <code>remote.log.storage.manager.class.name</code>
+  and <code>remote.log.storage.manager.class.path</code> to specify the implementation of RemoteStorageManager. One Apache 2.0 licensed implementation supporting all
+  three major hyperscalers' object stores (AWS, GCP, Azure) is <a href="https://github.com/Aiven-Open/tiered-storage-for-apache-kafka">available here</a>.
```

Comment:
I am not affiliated with Aiven in doing this, but I believe it's of service to the community to link to the only production grade plugin which supports all 3 major clouds.

lmk if you don't believe this is a good idea, i can be convinced to remove it (e.g if we have strict policies, etc)

Reply:
I don't think it's a good idea to mention the 3rd party project in the doc.

Reply:
@stanislavkozlovski , I think we should remove anything mentioning the 3rd party project in the doc.

Reply:
@stanislavkozlovski  afaik, we do not provide or mention any 3rd party projects in the docs. You can remove it in this PR. We can have a followup PR and discussion on it.